lib/handlers/compile.js

```diff
@@ -519,6 +511,7 @@ export class CompileHandler {
     }
 }
 
-export function SetTestMode() {
-    hasSetUpAutoClean = true;
+// If the interval is not finished, mocha will hang
+export function ClearIntervalForTestRunner() {
+    clearInterval(cleanInterval);
```

Comment:
This is not required; mocha supports a `--exit` flag for this scenario. Please let me know what you think about removing this and running the tests w/ the `--exit` flag.

Learn more at: https://mochajs.org/#-exit